pkg/files/testdata/filesWithDotDot/..one

```diff
@@ -0,0 +1 @@
+ignore
```

Comment:
To be more specific with the test you can create this and the other files with `..` prefix as symlinks. If I understand correctly that is the scenario that KubeArchive faces, isn't it?

Reply:
it is. I changed `..one`, `..two`, and `..three` to symlinks to their respective files without the `..`